src/functions/EmailRequestProcessor.ts

```diff
@@ -0,0 +1,33 @@
+import { S3Event, S3EventRecord, SQSRecord } from 'aws-lambda';
```

Comment:
Should this live in the services directory? Or maybe even a separate processor directory?

Reply:
I copied this pattern from cert-gen, what do you think is best? keep them the same or move into a folder?